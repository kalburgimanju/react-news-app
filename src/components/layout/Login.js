import React, { Component } from "react";

import LoginFormContainer from "../../containers/LoginFormContainer";

const propTypes = {};

const defaultProps = {};

class Login extends Component {
  render() {
    return (
      <section>
        <div className="wrapper">
          <LoginFormContainer />
        </div>
      </section>
    );
  }
}

Login.propTypes = propTypes;
Login.defaultProps = defaultProps;

export default Login;
