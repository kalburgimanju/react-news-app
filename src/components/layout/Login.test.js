import React from "react";
import Login from "./Login";

it("render Home", () => {
  const wrapper = shallow(<Login />);
  expect(wrapper).toMatchSnapshot();
});
