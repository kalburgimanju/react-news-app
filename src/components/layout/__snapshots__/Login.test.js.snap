// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Home 1`] = `
<section>
  <div
    className="wrapper"
  >
    <Connect(LoginFormContainer) />
  </div>
</section>
`;
